.cirrus.yml

```diff
@@ -48,8 +48,9 @@ env:
     #### Default to NOT operating in any special-case testing mode
     ####
     SPECIALMODE: "none"  # don't do anything special
-    TEST_REMOTE_CLIENT: false  # don't test remote client by default
-    ADD_SECOND_PARTITION: false  # will certainly fail inside containers
+    TEST_REMOTE_CLIENT: 'false'  # don't test remote client by default
+    ADD_SECOND_PARTITION: 'false'  # will certainly fail inside containers
+    MOD_LIBPOD_CONF: 'true'  # Update libpod.conf runtime if required by OS environment
```

Comment:
I don't believe we actually listen to `libpod.conf` for anything related to runtime. The test suite itself should be specifying runtime, manually, for each call to Podman.

Reply:
Actually I think @edsantiago system-tests do use it, no?

In any case, and since we want to (soonish) have both runc/crun happening on the same OS/version.  My thinking is it's best that automation lean toward over rather than under specifying this fact.

I'm open to arguments the other way though, as it certainly is an additional complication.

Reply:
Clarification: By over-specific I mean setting `/usr/share/containers/libpod.conf` ***AND*** setting `$OCI_RUNTIME` based on if cgroupsv2 is enabled in the kernel.